activerecord/lib/active_record/persistence.rb

```diff
@@ -403,13 +403,14 @@ def toggle!(attribute)
     #   # => #<Account id: 1, email: 'account@example.com'>
     #
     # Attributes are reloaded from the database, and caches busted, in
-    # particular the associations cache and the QueryCache.
+    # particular, the association's cache and the QueryCache.
```

Comment:
The old doc is correct. It's a cache of the associations, not a cache belonging to the associations. Comma change is fine 👍